### PR TITLE
Keep code style consistent - related to #639

### PR DESCRIPTION
**Pull request by zhx828**
To keep consistency, use underscore in if condition. @pieterlukasse could you review?

### CODE REVIEW

File: portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js [hunk 1]
```diff
@@ -257,7 +257,7 @@ var StudyViewProxy = (function() {
                 }else {
                     var cnaLength = obtainDataObject.arr.length;
                     for(var i = 0; i < cnaLength; i++) {
-                        if(obtainDataObject.arr[i]['MUTATION_COUNT']) {
+                        if(!_.isUndefined(obtainDataObject.arr[i]['MUTATION_COUNT'])) {
```
Comment (pieterlukasse) [inline portal/src/main/webapp/js/src/study-view/data/StudyViewProxy.js]: In line 258, should variable be called cnaLength or something like mutationLength? 

I would also advocate for keeping the same style in referring to attributes, so either ['MUTATION_COUNT'] OR .MUTATION_COUNT. Now you use both mixed up in this and next line.

**Comment by pieterlukasse** [comment]
@zhx828 : minor comment above

**Comment by zhx828** [comment]
@pieterlukasse Good catch, please see the changes in the commit above

**Comment by pieterlukasse** [comment]
@zhx828 OK!

**Pull request merged**